src/cursor-doc/token-cursor.ts

```diff
@@ -395,12 +407,17 @@ export class LispTokenCursor extends TokenCursor {
      * If possible, moves this cursor forwards past any whitespace, and then past the immediately following open-paren and returns true.
      * If the source does not match this, returns false and does not move the cursor.
      */
-    downList(): boolean {
+    downList(skipMetadata = false): boolean {
```

Comment:
Same remark here as with `forwardSexpr/backwardSexpr`, but the reciprocal move here would be `backwardUpList`, I think.

Reply:
Same response here as with forwardSexp 😄